common/components/StreetsSelector/index.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react'
+import React, { useState, useEffect, useRef } from 'react';
+
 import {Select} from "antd"
 
 import s from "@components/StreetsSelector/style.module.scss";
 
 const StreetsSelector = ({filters, setFilters, streets}) => {
+    const [dropdownWidth, setDropdownWidth] = useState('auto');
+    const selectRef = useRef(null);
+
+    useEffect(() => {
+        if (selectRef.current) 
+            setDropdownWidth(selectRef.current.offsetWidth);
+    }, []);
```

Comment:
This approach could broke on dropdown content change. As i can see there is no `dropdownWidth` update on dropdown content change